app/views/cases/overturned_shared/_new.html.slim

```diff
@@ -1,8 +1,9 @@
-= form_for @case, as: :case_overturned_sar, url: cases_path do |f|
+= form_for @case.object, as: "case_#{@correspondence_type_key}", url: cases_path do |f|
 
-  = hidden_field_tag 'correspondence_type', 'overturned_sar'
+  = hidden_field_tag 'correspondence_type', @correspondence_type_key
   = f.hidden_field :original_ico_appeal_id
-  = f.hidden_field :original_case_id, value: @case.original_ico_appeal.original_case_id
+  = f.hidden_field :original_case_id,
```

Comment:
Does this have to be on the page? I'm not seeing why we need it.

Reply:
These were just left-over from before, but you're right ... if we have the `original_ico_appeal`, we don't need and shouldn't have `original_case_id` so there's no risk of having them out-of-sync.